dojo/tools/crashtest_security/parser.py

```diff
@@ -48,13 +48,21 @@ def get_items(self, tree, test):
             if failure is None:
                 continue
 
-            title = node.get('name').upper()
+            title = node.get('name')
+            # Remove enumeration from title
+            title = re.sub(r' \([0-9]*\)', '', title)
```

Comment:
hmm.. I would probably be stricter on the regex here. This means that any number or hyphen within the whole title would be stripped out.

Reply:
Not exactly "anywhere" but well.. you I am sure get the point :) (not sure what crashtest titles look like, but assuming the worse is probably a good thing)

Reply:
It should not match the hyphen, but only numbers. Still, I will restrict it to the end of the string where this appears.